gui-apps/wdisplays/wdisplays-1.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+case "${PV}" in
+	"9999")
+		inherit git-r3
+		EGIT_REPO_URI="https://github.com/cyclopsian/wdisplays.git"
+		;;
+	*)
+		SRC_URI="https://github.com/cyclopsian/wdisplays/archive/${PV}.tar.gz -> ${P}.tar.gz"
+		KEYWORDS="~amd64"
+esac
+inherit meson xdg
+
+DESCRIPTION="GUI display configurator for wlroots compositors"
+HOMEPAGE="https://cyclopsian.github.io/wdisplays"
+
+BDEPEND="
+	x11-libs/gtk+:3[wayland]
+	gui-libs/wlroots"
+DEPEND="${BDEPEND}"
```

Comment:
Don't mix DEPEND with BDEPEND. And you're missing RDEPEND

https://devmanual.gentoo.org/general-concepts/dependencies/index.html